### PR TITLE
Adding README for cli 

### PR DESCRIPTION
**Pull request by alfasf**
# Summary

Fixes issue #470

Here's a description of changes:

* Adding description of class.
*Adding README style.
*No images included 

_Before you merge, make sure you visually check your changes in the deploy preview. A link to the preview will appear in the PR comments when ready._

cc @gdisf/admins

### CODE REVIEW

**Comment by verythorough** [comment]
Deploy preview ready!

Built with commit 70f3ad02faf156c320d14f3f0fd6c8512ed17809

https://deploy-preview-525--teaching-materials.netlify.com

**Comment by trucnguyen** [comment]
@alfasf Can you add the link in ./index.html? 
screen shot 2017-08-06 at 1 11 45 pm

Also, did you intend for the JSOO commits to be part of this PR as well?

File: jsoo/README.md [hunk 1]
```diff
@@ -1,31 +1,40 @@
 # Object-Oriented JavaScript
```
Comment (trucnguyen) [inline jsoo/README.md]: This should be in a separate PR?

File: cli/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Intro to the Command Line/Terminal
```
Comment (trucnguyen) [inline cli/README.md]: Please add a link to this on the index page in row PROG 101. Thanks

File: jsoo/README.md (hunk 1 above)
Comment (alfasf) [inline jsoo/README.md]: yes, for some reason the jsoo commit is still pushing but it should be closed.

File: cli/README.md (hunk 2 above)
Comment (alfasf) [inline cli/README.md]: I'm sorry trucnguyen. I'm still very confused on this reference. Are you asking to link the readme on index.html, or the index.html on readme? If so, how to reference it? Wish I could be further help.

Comment (trucnguyen) [inline cli/README.md]: Link the readme on the index page that lists all the courses.

Comment (trucnguyen) [inline cli/README.md]: On this row https://github.com/gdisf/teaching-materials/pull/525#issuecomment-320529609
image

**Comment by alfasf** [comment]
I think it's done. I saw my confusion. I didn't realize there was an index.html outside the dir cli which also contains another index.html so I didn't know where to link it.

**Comment by trucnguyen** [comment]
Perfect, thanks! To make sure old commits don't get included, did you try fetching upstream and merging upstream/master?

**Comment by alfasf** [comment]
yes i did.
